mkdocs.yml

```diff
@@ -269,15 +244,15 @@ markdown_extensions:
 
 
 ### Customization
-extra:
+# extra:
```

Comment:
You can use variables from the global scope to reuse in anywhere in the documentation. Variables defined by the user under `extra` are available globally.